libs/auth/src/drivers/interfaces/auth-service.interface.ts

```diff
@@ -0,0 +1,23 @@
+import { Observable } from 'rxjs';
+import { DaffAccountRegistration } from '../../models/account-registration';
+
+export interface DaffAuthServiceInterface<T extends DaffAccountRegistration> {
```

Comment:
This interface should be more generic, in case the app dev wants to customize the login request/response info or the registration request/response info. Like:
```
export interface DaffAuthServiceInterface<T extends DaffAccountLogin, V extends DaffLoginResponse, S extends DaffAccountRegistration, U extends DaffAccountRegistrationResponse> {
  /**
   * Logs the user in.
   *
   * @param {T} loginInfo
   * @returns {Observable<V>} An access token.
   * @memberof DaffAuthServiceInterface
   */
  login(loginInfo: T): Observable<V>;
  /**
   * Registers an account for the specified customer.
   *
   * @param {S} registration
   * @returns {Observable<U>} An access token.
   * @memberof DaffAuthServiceInterface
   */
  register(registration: S): Observable<U>;
}
```
This change will also effect your actions, effects, etc. which will also need to be generic.